deployment/7_upgrade_ens.js

```diff
@@ -0,0 +1,31 @@
+const TestENSRegistry = require('../build/TestENSRegistry');
+const TestReverseRegistrar = require('../build/TestReverseRegistrar');
+const ENSResolver = require('../build/ENSResolver');
+const ENSManager = require('../build/ENSManager');
+
+const deploy = async (network) => {
+    const config = configurator.config;
+    const domain = config.ENS.domain;
+    const deploymentAccount = await deployer.signer.getAddress();
+
+	// ENS domain
+	const parts = domain.split('.');
+	const extension = parts[1];
+	const domainName = parts[0];
+
+    // Instantiate the ENS Registry
+    const ensRegistry = await deployer.wrapDeployedContract(TestENSRegistry, config.ENS.ensRegistry);
+
+	// Create the 'argentx.xyz' wallet ENS namespace
+	const setSubnodeOwnerArgent = await ensRegistry.contract.setSubnodeOwner(utils.namehash(extension), utils.sha3(domainName), deploymentAccount);
+	await ensRegistry.verboseWaitForTransaction(setSubnodeOwnerArgent, `Setting Subnode Owner for ${domainName}.${extension}`);
+
+    const ensResolver = await deployer.deploy(ENSResolver);
```

Comment:
We do not need to deploy new versions of the `ENSResolver` and the `ReverseRegistrar` as they are not affected by the update.

Reply:
There's these changes https://github.com/argentlabs/argent-contracts/pull/54/files#diff-5f255ee701d7d41347fe8dac7c4b29e2R147

Reply:
I believe the `TestReverseRegistrar.sol` contract is only used for tests. On both Ropsten and Mainnet we use the official ReversRegistrar deployed by the ENS.

Reply:
Can we add config for it then, I believe it's at `0x084b1c3C81545d370f3634392De611CaaBFf8148` on both mainnet and ropsten